plugins/processors/topk/topk_test.go

```diff
@@ -411,7 +411,7 @@ func TestTopkGroupbyMetricName1(t *testing.T) {
 	topk.K = 1
 	topk.Aggregation = "sum"
 	topk.AddAggregateFields = []string{"value"}
-	topk.GroupBy = []string{}
+	topk.GroupBy = make([]string, 0)
```

Comment:
Remove.

Reply:
Needs to be there. It overrides value initialized in constructor.